roland_cutstudio.py

```diff
@@ -388,21 +393,28 @@ def output(array):
     cmd = [INKSCAPEBIN, "-T", "--export-ignore-filters",  "--export-area-drawing", "--export-filename="+filename+".inkscape.eps", filename+".filtered.svg"]
 inkscape_eps_file = filename + ".inkscape.eps"
 
-#print(" ".join(cmd), file=sys.stderr)
+#inkex.utils.debug(" ".join(cmd), file=sys.stderr)
 assert 0 == subprocess.call(cmd, stderr=DEVNULL), 'EPS conversion failed: command returned error: ' + '"' + '" "'.join(cmd) + '"'
 assert os.path.exists(inkscape_eps_file), 'EPS conversion failed: command did not create result file: ' + '"' + '" "'.join(cmd) + '"' 
 
 EPS2CutstudioEPS(inkscape_eps_file, filename+".cutstudio.eps", mirror=("--mirror=true" in sys.argv))
 
 if os.name=="nt":
     DETACHED_PROCESS = 8 # start as "daemon"
-    Popen([which("CutStudio\CutStudio.exe"), "/import", filename+".cutstudio.eps"], creationflags=DETACHED_PROCESS, close_fds=True)
-else:
-    #raise Exception("CutStudio on Mac and on Linux-Wine not yet supported, please open cutstudio yourself.")
-    print("Your file was saved to:\n" + filename+".cutstudio.eps" + "\n Please open that with CutStudio.", file=sys.stderr)
-    # Popen(["inkscape", filename+".filtered.svg"], stderr=DEVNULL)
-    #Popen(["inkscape", filename+".cutstudio.eps"])
-    pass
+    Popen([which("CutStudio\CutStudio.exe", True), "/import", filename+".cutstudio.eps"], creationflags=DETACHED_PROCESS, close_fds=True)
+else: #check if we have access to "wine"
+    if which("wine", False) is not None:
+        if which("/opst/CutStudio/CutStudio.exe", False) is not None:
+            with os.popen("wine /opt/CutStudio/CutStudio.exe /import T:\\\\" + filename.split('/tmp/')[1] + ".cutstudio.eps", "r") as cutstudio:
+                result = cutstudio.read()
+        else:
+            inkex.utils.debug("Found a wine installation on your system but no CutStudio.exe. You can easily emulate this Windows application on Linux using wine. To do this provide a valid CutStudio installation in directory \"/opt/CutStudio/\". You may create a symlink with the shell command \"ln -sf /your/path/to/cutstudio /opt/CutStudio\" to point to this dir, or just edit the InkScape extension source code. As second step you need to configure the drive letter T: to point to the directory \"/tmp/\". This is done with a few mouse clicks within the \"winecfg\" utility. The wine emulation was tested to work properly with Roland CutStudio version 3.10. For now your file was saved to:\n" + filename + ".cutstudio.eps")
```

Comment:
- Maybe flip the message: First print the filename, then the long text?
- Maybe also show it on Linux if wine is not installed, since it's easy to install?
- Why `/opt`? It is usually not user-writable, so you'll have to create the symlink as root. We could simply use the default install directory: `$HOME/.wine/dosdevices/c:/Progr*/CutStudio/CutStudio.exe` Also, this path should be *one* variable, not two places in the script.
- For extra comfort, we could also copy the file to `$HOME/.wine/dosdevices/c:/temp.cutstudio.eps` or similar. So the T: drive is not required.